README.md

```diff
@@ -59,6 +59,7 @@ export default withSnackbar()(MyComponent) // export the wrapped component
 * `message` (string) – message to display
 * `action` (string, _optional_) – label for the action button
 * `handler` (function, _optional_) – click handler for the action button
+* `close` (boolean, _optional_) – Whether to show the close button or not
```

Comment:
:thinking: I think we'll be more future-proof if we made the fourth argument an `options` object, with a `closeable` option.